checks/cs50/2017/x/mario/mario.py

```diff
@@ -0,0 +1,76 @@
+import os
+import sys
+
+sys.path.append(os.getcwd())
+from check50 import *
+
+class Mario(Checks):
+    @check()
+    def exists(self):
+        """mario.c exists."""
+        super(Mario, self).exists("mario.c")
```

Comment:
Did we decide this is the optimal Python 2/3 approach? I worry this is an API detail that a TF will always have to look up.